scripts/core/exp-command-buffer.yml

```diff
@@ -0,0 +1,246 @@
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Command Buffer Extension APIs"
+ordinal: "0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Command Buffer Desc Type"
+name: $x_exp_command_buffer_desc_t
+base: $x_base_desc_t
+members: 
+--- #--------------------------------------------------------------------------
+type: typedef
+desc: "A value that identifies a command inside of a command buffer, used for defining dependencies between commands."
+class: $xCommandBufferExp
+name: $x_exp_command_buffer_sync_point_t
+value: "uint32_t"
+--- #--------------------------------------------------------------------------
+type: handle
+desc: "Handle of Command Buffer object"
+class: $xCommandBufferExp
+name: "$x_exp_command_buffer_handle_t"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Create a Command Buffer object"
+class: $xCommandBuffer
+name: CreateExp
+decl: static
+ordinal: "0"
+details:
+    - "Create a command buffer object"
+params:
+    - type: $x_context_handle_t
+      name: hContext
+      desc: "[in] handle of the context object"
+    - type: $x_device_handle_t
+      name: hDevice
+      desc: "[in] handle of the device object"
+    - type: "const $x_exp_command_buffer_desc_t*"
+      name: pCommandBufferDesc
+      desc: "[in][optional] CommandBuffer descriptor"
+    - type: "$x_exp_command_buffer_handle_t*"
+      name: phCommandBuffer
+      desc: "[out] pointer to Command Buffer handle"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get a reference to a command buffer handle. Increment the command buffer object's reference count."
```

Comment:
"Get a reference to a command buffer handle." could maybe remove this starting sentence, "get" makes me think the entry-point is returning something, which it's not doing.